Source/Diffable Collection/FetchedCollectionDiffableDataSource.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  FetchedCollectionDataSource (Diffable).swift
+//  AppwiseCore
+//
+//  Created by Jonathan Provo on 14/03/2022.
+//
+
+import CoreData
+import UIKit
+
+@available(iOS 13.0, *)
+public typealias FetchedDiffableDataSource = UICollectionViewDiffableDataSource<FetchedDiffableSection, FetchedDiffableItem>
+
+@available(iOS 13.0, *)
+public final class FetchedCollectionDiffableDataSource: NSObject, NSFetchedResultsControllerDelegate {
+	// MARK: - Properties
+	/// A boolean indicating whether empty sections should be hidden or not. The default value is `false`.
+	public var isHidingEmptySections: Bool = false
```

Comment:
I'd move this to a separate PR, and add this to the older date sources as well. Seems like a useful behaviour to have.